enforceBeachball.js

```diff
@@ -0,0 +1,12 @@
+const cp = require('child_process');
+
+cp.exec('yarn beachball < killProcessCommand.txt', (err, stdout, stderr) => {
```

Comment:
Why does this need the killProcessCommand.txt file? What's in the file?

Reply:
I think this is a duplicate comment- will condense into the other thread.